### PR TITLE
fix: remove preventDefault in passive event listener

### PR DESCRIPTION
**Pull request by stop2stare**

### CODE REVIEW

File: src/Carousel.vue [hunk 1]
```diff
@@ -548,8 +536,6 @@ export default {
         return;
       }
 
-      // we are good to prevent the move and handle the translation
-      e.preventDefault();
```
Comment (stop2stare) [inline src/Carousel.vue]: when this line was written, maybe the author was still using chrome 56-, and option passive defaults to false.

**Comment by quinnlangille** [comment]
Hmmm, I don't know if removing that line will achieve the same results. I think we still want our listeners to be `passive: false` here if we expect developers to be modifying the scroll behaviour.

**Comment by stop2stare** [comment]
@quinnlangille sure it'll achieve the same results. Please check out [this chromestatus documentation](https://www.chromestatus.com/feature/5093566007214080) and [this addEventListener documentation](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener). The reason why warnings occurred in console was that we invoked preventDefault in passive event listener. The former PR set `passive` to false, and that resolved this warning.
But, why does Chrome set this option to true?

> This change prevents the listener from blocking page rendering while a user is scrolling.

The detailed documentation is [here](https://developers.google.com/web/updates/2017/01/scrolling-intervention)
SO, manually setting option `passive` to false is not the best choice, and even worsening the scrolling performances.
BUT, maybe you or @toddlawton wrote this line for some other special reasons or optimization. This PR just offers you guys another way to solve this problem.😄

**Comment by quinnlangille** [comment]
Ok I agree, I'll merge this in and release with the next batch of updates! Thanks @stop2stare

**Pull request merged**